### PR TITLE
Added Mult-Handlers support and some useful functions

### PR DESCRIPTION
**Pull request by OGFris**

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -41,14 +41,18 @@ var (
...
+
 type Server struct {
-	Logger         raknet.Logger
-	Handler        Handler // TODO: supports multiple handlers
+	Logger         utils.Logger
+	Handlers        Handlers
 	MaxConnections int
 	MTU            int
-	Identifier     identifier.Identifier
+	Identifier     identifier.Minecraft
```
Comment (beito123) [inline server/server.go]: go-raknet is just a raknet server library.
If it changes, goraknet will be dedicated for minecraft.

Comment (beito123) [reply server/server.go]: Others are very good job!

Comment (OGFris) [reply server/server.go]: yeah i guess you're right

Comment (beito123) [reply server/server.go]: Could you commit a fixed code to the PR?

Comment (beito123) [reply server/server.go]: or do you want me to commit?

**Comment by beito123** [comment]
Thank you!

**Pull request merged**